dags/error_aggregates.py

```diff
@@ -1,29 +1,39 @@
-import datetime
+# Generated via query_scheduling/generate_airflow_dags
 
-from airflow import models
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
```

Comment:
`ExternalTaskSensor` isn't used here.  Is this just an artifact of the dag generation?

Reply:
It is. Currently, the imports are hard-coded into the DAG template. I guess as a future improvement it could make sense to keep track of whether the imports are actually used.